webrtc.html

```diff
@@ -7778,6 +7785,10 @@ <h3><dfn>RTCRtpTransceiver</dfn> Interface</h3>
           <p>Let <var>transceiver</var> have a <dfn>[[\Receiver]]</dfn> internal
           slot, initialized to <var>receiver</var>.</p>
         </li>
+        <li data-tests="RTCPeerConnection-addTransceiver.https.html">
+          <p>Let <var>transceiver</var> have a <dfn>[[\Mid]]</dfn> internal
+          slot, initialized to <var>null</var>.</p>
+        </li>
```

Comment:
An internal slot is good. But we have existing language (i.e. when setting an RTCSessionDescription) that says to "Set transceiver's mid value to the mid of the media description" and similar sentences in several places. Update those to modify the internal slot.